pkgs/applications/version-management/git-and-tools/git-team/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = "sha256-xJMWPDuqoNtCCUnKuUvwlYztyrej1uZttC0NsDvYnXI=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    go run main.go --generate-man-page > $out/${pname}.1
+    installManPage $out/${pname}.1
```

```suggestion
    installManPage <(go run main.go --generate-man-page)
```

---

While I generally like the idea, this is actually failing on my machine during build.

>@nix { "action": "setPhase", "phase": "installPhase" }
installing
installManPage: error: path missing manpage section suffix: /dev/fd/63

---

🤔 Then it might not work with installManPage. I am not sure actually. Anyway we don't want to pipe it to $out/*.1.

```suggestion
    go run main.go --generate-man-page > ${pname}.1
    installManPage ${pname}.1
```

---

Updated